view/adminhtml/templates/System/Config/customAttributes.phtml

```diff
@@ -88,13 +88,13 @@
                     $_row->setAction('');
                     $_row->setActionClass('hidden');
                 }
-                echo "arrayAttributes{$block->escapeHtml($_htmlId)}.add(" . $_row->toJson() . ");\n";
+                echo $escaper->escapeHtml("arrayAttributes{$_htmlId}.add(" . $_row->toJson() . ");\n");
             }
             ?>
 
             // Toggle the grid availability, if element is disabled (depending on scope)
             <?php if ($block->getElement()->getDisabled()):?>
-                toggleValueElements({checked: true}, $('grid<?= $block->escapeHtml($_htmlId) ?>').parentNode);
+                toggleValueElements({checked: true}, $('grid<?= $escaper->escapeHtml($_htmlId) ?>').parentNode);
```

Comment:
Just a question: isn't here anything better like `$escaper->escapeJs()`? Normally `escapeJs` is stricter than `escapeHtml` which only escapes HTML, but not the quotes or other code-breaking chars.

Reply:
`escapeJs` ?

Reply:
Ok, I'll try it